### PR TITLE
Css three column layout

### PR DESCRIPTION
**Pull request by arjunkahlon**
Screen Shot 2022-07-01 at 12 26 01 AM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  font-size: 22px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.column {
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:30]: `column` what? How big is this column? Details matter 🤓

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+#page-header {
+  color: white;
+  font-family: Lato, sans-serif;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0;
+  font-size: 22px;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:22]: I would double-check that all these declarations are actually necessary. This isn't a flexbox, so `justify-content` and `align-items` are likely unnecessary.

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+.column {
+  flex-basis: calc(100%/3);
+}
+
+.feature-section {
+  color: white;
+  display: flex;
+  flex-direction: column;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:37]: Is this necessary?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css (hunk 1 above)
Comment (arjunkahlon) [reply css-three-column-layout/styles.css]: Rewrote the assignment, and made sure to name the column 'column-third' as it occupies a third of the container row.

File: css-three-column-layout/styles.css (hunk 2 above)
Comment (arjunkahlon) [reply css-three-column-layout/styles.css]: Rewrote the assignment, and made sure to only use flexbox properties when applicable. I created a .center class for flexbox items and a text-center class for text elements.

File: css-three-column-layout/styles.css (hunk 3 above)
Comment (arjunkahlon) [reply css-three-column-layout/styles.css]: I have a better understanding of flexbox now and see why this implementation was incorrect.

**Comment by arjunkahlon** [comment]
Hey Cody, I rewrote the assignment based on your feedback. I think I have a much better understanding on flexbox now. Here is an attached screenshot of the new web page. 

Screen Shot 2022-07-01 at 3 28 35 PM (2)

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**